web/src/user_group_edit.ts

```diff
@@ -1296,12 +1387,13 @@ export function setup_page(callback: () => void): void {
         };
 
         const groups_overlay_html = render_user_group_settings_overlay(template_data);
-
         const $groups_overlay_container = scroll_util.get_content_element(
             $("#groups_overlay_container"),
         );
         $groups_overlay_container.html(groups_overlay_html);
-
+        update_displayed_groups(
+            z.string().optional().parse(ls.get(user_group_visibility_key)) ?? FILTERS.ALL_GROUPS,
```

Comment:
What is the default value of the filter? Here `ALL_GROUPS` is passed but in `user_group_settings_overlay.hbs`, `hide-deactived-user-groups` class is added. I think would be easier to handle it in JS since the value is stored in localstorage, so the class added in template file can be removed.